### PR TITLE
EN_Reboot: class descriptions in class counter NPC

### PR DESCRIPTION
**Pull request by Gunvarrel39**
only hunter was translated so I translated the other five.

### CODE REVIEW

File: Translated/UI/ui_mainmode.csv [hunk 1]
```diff
@@ -1590,11 +1590,11 @@ Name#0,"""Character Name"""
...
-RangerExplanation#0,"""<yellow>中距離戦闘向きのクラス<c><br>射撃武器により、エネミーと一定の距離を置いて戦う。<br>特殊弾で、他のプレイヤーを支援することもできる。"""
-ForceExplanation#0,"""<yellow>遠距離戦闘向きのクラス<c><br>遠距離から、“テクニック”でエネミーを攻撃する。<br>多彩なテクニックで、エネミーの弱点属性を巧みに狙う。"""
-FighterExplanation#0,"""<yellow>超近距離戦闘向きのクラス<c><br>エネミーと肉薄する距離での戦闘を得意とする。<br>防御力を犠牲にした強力な猛攻撃で、エネミーを粉砕する。"""
-GunnerExplanation#0,"""<yellow>近距離戦闘向きのクラス<c><br>射撃武器による近接戦闘を得意とする。<br>すさまじい弾幕と優れた機動力で、エネミーを翻弄する。"""
-TecherExplanation#0,"""<yellow>近距離および遠距離戦闘向きのクラス<c><br>“テクニック”に長け、他のプレイヤーを支援する。<br>一時的な能力強化や回復などで、戦闘を有利に進める。"""
+RangerExplanation#0,"""<yellow>Medium Ranged Class<c><br>Fight while keeping distance from enemies with a ranged weapon.<br>You can also support other players with special bullets."""
+ForceExplanation#0,"""<yellow>Long Ranged Class<c><br>Attack enemies from long ranges using \""Techniques\"".<br>Harass enemies' elemental weaknesses by using various Techniques."""
+FighterExplanation#0,"""<yellow>Ultra-close Ranged Class<c><br>Close in and take the fight personally with enemies.<br>Crush your enemies with powerful, fierce attacks at the expense of your defense."""
```
Comment (DabirSA) [inline Translated/UI/ui_mainmode.csv]: "Take the fight personally with enemies" sounds like you're offended by them fighting back or something. I get what you're going for but there must be a better way to word that.

File: Translated/UI/ui_mainmode.csv [hunk 2]
```diff
@@ -1590,11 +1590,11 @@ Name#0,"""Character Name"""
 Guild#0,"""Team Name"""
 NoGuild#0,"""Teamless"""
 HunterExplanation#0,"""<yellow>Close Ranged Class<c><br>A class whose value lies in close-range combat. Has high HP and defense, and is excellent at fighting single targets."""
-RangerExplanation#0,"""<yellow>中距離戦闘向きのクラス<c><br>射撃武器により、エネミーと一定の距離を置いて戦う。<br>特殊弾で、他のプレイヤーを支援することもできる。"""
-ForceExplanation#0,"""<yellow>遠距離戦闘向きのクラス<c><br>遠距離から、“テクニック”でエネミーを攻撃する。<br>多彩なテクニックで、エネミーの弱点属性を巧みに狙う。"""
-FighterExplanation#0,"""<yellow>超近距離戦闘向きのクラス<c><br>エネミーと肉薄する距離での戦闘を得意とする。<br>防御力を犠牲にした強力な猛攻撃で、エネミーを粉砕する。"""
-GunnerExplanation#0,"""<yellow>近距離戦闘向きのクラス<c><br>射撃武器による近接戦闘を得意とする。<br>すさまじい弾幕と優れた機動力で、エネミーを翻弄する。"""
-TecherExplanation#0,"""<yellow>近距離および遠距離戦闘向きのクラス<c><br>“テクニック”に長け、他のプレイヤーを支援する。<br>一時的な能力強化や回復などで、戦闘を有利に進める。"""
+RangerExplanation#0,"""<yellow>Medium Ranged Class<c><br>Fight while keeping distance from enemies with a ranged weapon.<br>You can also support other players with special bullets."""
+ForceExplanation#0,"""<yellow>Long Ranged Class<c><br>Attack enemies from long ranges using \""Techniques\"".<br>Harass enemies' elemental weaknesses by using various Techniques."""
```
Comment (DabirSA) [inline Translated/UI/ui_mainmode.csv]: I don't like "harrass" here.

File: Translated/UI/ui_mainmode.csv [hunk 3]
```diff
@@ -1590,11 +1590,11 @@ Name#0,"""Character Name"""
 Guild#0,"""Team Name"""
 NoGuild#0,"""Teamless"""
 HunterExplanation#0,"""<yellow>Close Ranged Class<c><br>A class whose value lies in close-range combat. Has high HP and defense, and is excellent at fighting single targets."""
-RangerExplanation#0,"""<yellow>中距離戦闘向きのクラス<c><br>射撃武器により、エネミーと一定の距離を置いて戦う。<br>特殊弾で、他のプレイヤーを支援することもできる。"""
-ForceExplanation#0,"""<yellow>遠距離戦闘向きのクラス<c><br>遠距離から、“テクニック”でエネミーを攻撃する。<br>多彩なテクニックで、エネミーの弱点属性を巧みに狙う。"""
-FighterExplanation#0,"""<yellow>超近距離戦闘向きのクラス<c><br>エネミーと肉薄する距離での戦闘を得意とする。<br>防御力を犠牲にした強力な猛攻撃で、エネミーを粉砕する。"""
-GunnerExplanation#0,"""<yellow>近距離戦闘向きのクラス<c><br>射撃武器による近接戦闘を得意とする。<br>すさまじい弾幕と優れた機動力で、エネミーを翻弄する。"""
-TecherExplanation#0,"""<yellow>近距離および遠距離戦闘向きのクラス<c><br>“テクニック”に長け、他のプレイヤーを支援する。<br>一時的な能力強化や回復などで、戦闘を有利に進める。"""
+RangerExplanation#0,"""<yellow>Medium Ranged Class<c><br>Fight while keeping distance from enemies with a ranged weapon.<br>You can also support other players with special bullets."""
```
Comment (DabirSA) [inline Translated/UI/ui_mainmode.csv]: Ranger has *two* ranged weapons, so "*a* ranged weapon" reads a bit weirdly. Also, "keeping your distance" would scan better.

File: Translated/UI/ui_mainmode.csv [hunk 4]
```diff
@@ -1590,11 +1590,11 @@ Name#0,"""Character Name"""
...
+ForceExplanation#0,"""<yellow>Long Ranged Class<c><br>Attack enemies from long ranges using \""Techniques\"".<br>Harass enemies' elemental weaknesses by using various Techniques."""
+FighterExplanation#0,"""<yellow>Ultra-close Ranged Class<c><br>Close in and take the fight personally with enemies.<br>Crush your enemies with powerful, fierce attacks at the expense of your defense."""
+GunnerExplanation#0,"""<yellow>Close Ranged Class<c><br>Good at close-quarters combat with a ranged weapon.<br>Unleash fierce barrages and utilize your mobility to bamboozle enemies."""
+TecherExplanation#0,"""<yellow>Close/Long Ranged Class<c><br>Support other players using \""Techniques\"".<br>Shift the battle in your favor by buffing and healing your teammates."""
```
Comment (DabirSA) [inline Translated/UI/ui_mainmode.csv]: Don't need quotes on Techniques. Also, "Shift the battle *to* your favour" would probably read a bit better.

File: Translated/UI/ui_mainmode.csv [hunk 5]
```diff
@@ -1590,11 +1590,11 @@ Name#0,"""Character Name"""
...
-ForceExplanation#0,"""<yellow>遠距離戦闘向きのクラス<c><br>遠距離から、“テクニック”でエネミーを攻撃する。<br>多彩なテクニックで、エネミーの弱点属性を巧みに狙う。"""
-FighterExplanation#0,"""<yellow>超近距離戦闘向きのクラス<c><br>エネミーと肉薄する距離での戦闘を得意とする。<br>防御力を犠牲にした強力な猛攻撃で、エネミーを粉砕する。"""
-GunnerExplanation#0,"""<yellow>近距離戦闘向きのクラス<c><br>射撃武器による近接戦闘を得意とする。<br>すさまじい弾幕と優れた機動力で、エネミーを翻弄する。"""
-TecherExplanation#0,"""<yellow>近距離および遠距離戦闘向きのクラス<c><br>“テクニック”に長け、他のプレイヤーを支援する。<br>一時的な能力強化や回復などで、戦闘を有利に進める。"""
+RangerExplanation#0,"""<yellow>Medium Ranged Class<c><br>Fight while keeping distance from enemies with a ranged weapon.<br>You can also support other players with special bullets."""
+ForceExplanation#0,"""<yellow>Long Ranged Class<c><br>Attack enemies from long ranges using \""Techniques\"".<br>Harass enemies' elemental weaknesses by using various Techniques."""
+FighterExplanation#0,"""<yellow>Ultra-close Ranged Class<c><br>Close in and take the fight personally with enemies.<br>Crush your enemies with powerful, fierce attacks at the expense of your defense."""
+GunnerExplanation#0,"""<yellow>Close Ranged Class<c><br>Good at close-quarters combat with a ranged weapon.<br>Unleash fierce barrages and utilize your mobility to bamboozle enemies."""
```
Comment (DabirSA) [inline Translated/UI/ui_mainmode.csv]: Again, Gunner has multiple weapon types and its main weapon type is also multiple weapons. "weapons" plural, please. Also, "bamboozle" is a great word but feels slightly inappropriate here. Maybe something like "confound" or "elude"?

**Review by DabirSA — CHANGES_REQUESTED** [review]

File: Translated/UI/ui_mainmode.csv (hunk 1 above)
Comment (Gunvarrel39) [reply Translated/UI/ui_mainmode.csv]: Thanks, I'll try to reword it. My translation style is kinda liberal so I'll try to come up with something closer to the original meaning.

File: Translated/UI/ui_mainmode.csv (hunk 4 above)
Comment (Gunvarrel39) [reply Translated/UI/ui_mainmode.csv]: I googled it up and it seems that the correct grammar is indeed _in_ your favor, so I'll leave this one unchanged.

File: Translated/UI/ui_mainmode.csv (hunk 5 above)
Comment (Gunvarrel39) [reply Translated/UI/ui_mainmode.csv]: Thanks again. Changed the verb to daze- its meaning seems to be fitting.

**Comment by Gunvarrel39** [comment]
I just realized that the script has pretty much been translated & commited by SynthSy a couple hours ago, so feel free to delete this pull request or use mine- whichever you think works better.

**Pull request closed without merging**